src/components/KPop/KPop.vue

```diff
@@ -275,15 +221,21 @@ export default defineComponent({
       type: Number,
       default: 1000,
     },
+    hideCloseIcon: {
+      type: Boolean,
+      default: false,
+    },
   },
-  emits: ['opened', 'closed'],
+  emits: ['open', 'close', 'popover-click'],
```

Comment:
question: any reason to use `popover-click` instead of just `click`?

Reply:
I think that just `click` would be confusing as component essentially consists of two parts: trigger and popover.